x/stake/genesis.go

```diff
@@ -1,17 +1,19 @@
 package stake
 
 import (
+	abci "github.com/tendermint/tendermint/abci/types"
+	tmtypes "github.com/tendermint/tendermint/types"
+
 	sdk "github.com/cosmos/cosmos-sdk/types"
 	"github.com/cosmos/cosmos-sdk/x/stake/types"
-	tmtypes "github.com/tendermint/tendermint/types"
 )
 
 // InitGenesis sets the pool and parameters for the provided keeper and
 // initializes the IntraTxCounter. For each validator in data, it sets that
 // validator in the keeper along with manually setting the indexes. In
 // addition, it also sets any delegations found in data. Finally, it updates
 // the bonded validators.
-func InitGenesis(ctx sdk.Context, keeper Keeper, data types.GenesisState) {
+func InitGenesis(ctx sdk.Context, keeper Keeper, data types.GenesisState) (res []abci.Validator) {
```

Comment:
function comment should be updated to reflect these changes.

If I understand this correctly we want to pass back the abci validators so that we do not need to have the validators twice in the genesis file? Tendermint will just read the validators from response init chain?